public/css/mixin/card.less

```diff
@@ -19,5 +19,20 @@
     .card-body {
       padding: .5em;
     }
+
+    .card-footer {
+      display: flex;
+      justify-content: center;
+      padding: 0.1em;
+      border-top: 1px solid @gray-light;
+
+      .key {
+        width: 9.2em;
+      }
+
+      .value {
+        font-weight: bold;
+      }
+    }
```

Comment:
* This now also applies to the tactical overview. Don't place it in the mixin but in the class for the check-statistics widget
* Remove the padding, the horizontal key value already has one and should be enough
* Use `width: auto` for the key and add a `margin-right: .28125em; // calculated &nbsp; width`
  * This ensures that no matter how the key is localized, it only requires that much space it really needs